Podfile

```diff
@@ -8,7 +8,7 @@ platform :ios, '13.0'
 target 'RudderFirebase' do
     project 'RudderFirebase.xcodeproj'
     pod 'Rudder', '~> 2.0'
-    pod 'FirebaseAnalytics', '9.2.0'
+    pod 'FirebaseAnalytics', '10.28.0'
```

Comment:
I think we don't need this. Can we please remove this `pod 'FirebaseAnalytics', '10.28.0'`?

Reply:
Okay,I have removed this line